source/server/http/admin.cc

```diff
@@ -316,7 +316,13 @@ Http::Code AdminImpl::handlerStats(const std::string& url, Buffer::Instance& res
 std::string AdminImpl::formatTagsForPrometheus(const std::vector<Stats::Tag>& tags) {
   std::vector<std::string> buf;
   for (const Stats::Tag& tag : tags) {
-    buf.push_back(fmt::format("{}={}", tag.name_, tag.value_));
+    std::string label_value = tag.value_;
+    std::string label_name = tag.name_;
+
+    std::replace(label_value.begin(), label_value.end(), '.', '_');
```

Comment:
I think this would be slightly better if you made a small helper function called `sanitizePrometheusName` or something like that, which then returns an std::string, since then you could do it all inline with the format call like you did before. It would also hide the replacements in case we need to make more in the future.

Reply:
Got it. Let me try that and resubmit.